lighthouse-core/test/gather/gatherers/full-page-screenshot-test.js

```diff
@@ -7,12 +7,22 @@
 
 /* eslint-env jest */
 
-const {
-  createMockContext,
-  mockDriverSubmodules,
-} = require('../../fraggle-rock/gather/mock-driver.js');
+import {createMockContext, mockDriverSubmodules} from '../../fraggle-rock/gather/mock-driver.js';
+// import FullPageScreenshotGatherer from '../../../gather/gatherers/full-page-screenshot.js';
```

Comment:
Any you purposefully keeping these as comments?

Reply:
Yes, please see my original post